src/pages/submit/index.tsx

```diff
@@ -37,7 +40,11 @@ export default function NewProposalPage() {
           <CategoryBanner type={ProposalType.POI} onClick={() => setShowPOIProposalModal(true)} />
           <CategoryBanner type={ProposalType.BanName} href={locations.submit(ProposalType.BanName)} />
           <CategoryBanner type={ProposalType.LinkedWearables} href={locations.submit(ProposalType.LinkedWearables)} />
-          <CategoryBanner type={ProposalType.Grant} href={locations.submit(ProposalType.Grant)} />
+          <CategoryBanner
+            type={ProposalType.Grant}
+            href={locations.submit(ProposalType.Grant)}
+            active={!isGrantProposalSubmitEnabled(NOW)}
```

Comment:
Shouldn't this be active if the submit is enabled?

Reply:
Yes, wrong date 😆